guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -219,7 +219,7 @@ ifdef::satellite[]
 endif::[]
 . Change SELinux to enforcing mode.
 For more information, see link:{RHELDocsBaseURL}9/html/upgrading_from_rhel_8_to_rhel_9/applying-security-policies_upgrading-from-rhel-8-to-rhel-9#changing-selinux-mode-to-enforcing_applying-security-policies[Changing SELinux mode to enforcing] in _Upgrading from RHEL{nbsp}8 to RHEL{nbsp}9_.
-ifeval::["{mode}" == "disconnected"]
+ifeval::["{mode}" != "disconnected"]
```

Comment:
I thought we wanted to include this instruction in all cases? @evgeni

Reply:
In disconnected, there is no sub man :)

Reply:
https://www.youtube.com/watch?v=lg7MAacSPNM